benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -302,6 +312,19 @@ class KeycloakScenarioBuilder {
       })
   }
 
+  def createRealm(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(http("Create realm")
+        .post(REALMS_ENDPOINT)
+        .header("Authorization", "Bearer ${token}")
+        .header("Content-Type", "application/json")
+        .body(StringBody("{\"id\":\"realm-${gatlingUniqueUserId}\",\"realm\":\"realm-${gatlingUniqueUserId}\",\"enabled\": true}"))
```

Comment:
As noted above: instead of `gatlingUniqueUserId` a UUID could be used here.

Reply:
Yep, see my previous comment

Reply:
@davoustp in #321 you can use the below example to be able to avoid inserting escape chars. that increases the readability of the json body here. reference : https://gatling.io/docs/gatling/reference/current/http/request/?#stringbody

```// with a static payload
http("name").post("/")
  .body(StringBody("""{ "foo": "staticValue" }"""))

// with a Gatling EL string payload
http("name").post("/")
  .body(StringBody("""{ "foo": "#{dynamicValue}" }"""))

// with a function payload
http("name").post("/")
  .body(StringBody(session => s"""{ "foo": "${session("dynamicValueKey").as[String]}" }"""))```

Reply:
Hi @kami619 
Ah, I knew there was some syntactic sugar which I could use to avoid the escaping, but could not find it from the top of my head. ;-)
This is much better now.
Fixed.